js/views/pageNavVw.js

```diff
@@ -131,6 +131,25 @@ module.exports = Backbone.View.extend({
       var languageList = new window.List('homeModal-languageList', {valueNames: ['homeModal-language'], page: 1000});
       self.initAccordion('.js-profileAccordion');
 
+      // pre-select lauguage and scroll it to the top.
+      var localLanguage = window.navigator.language;
+      var currentLanguage = null;
+      $('.js-homeModal-languageSelect').each(function () {
+        if(this.getAttribute("data-code") == localLanguage) {
+          currentLanguage = this;
+          return false;
+        }
+      });
+      if(currentLanguage != null) {
+        $("#" + localLanguage).prop('checked', true);
```

Comment:
For the languages, you can't just set the checked property to true. If you test this branch, you'll see that both English and the currentLanguage are both checked. This is because a "selected" value is set on the model on line 271, equal to the language in the model. 

Move the part of this code that gets the local language to the initialize function, and use it to set the language in the model. That will automatically set checked property of the right language in the languageList view. 

The part of the code that scrolls the selected language into view should stay here. 

That should also solve another issue. When the language is selected, the view is re-rendered. If the local language is changed in the accordionReady function, the user's selected language will be reset to the local language. By moving that code to the initialize function, it will only happen once.